omod/src/main/java/org/bahmni/module/hip/web/service/ExistingPatientService.java

```diff
@@ -25,7 +25,13 @@ public ExistingPatientService(PatientService patientService, ExistingPatientDao
     }
 
     public List<Patient> getMatchingPatients(String phoneNumber) {
-        return existingPatientDao.getPatientsWithPhoneNumber(phoneNumber);
+        String phoneNumberWithOutCode = phoneNumber.substring(phoneNumber.length() - 10);
+        String []formats = new String[4];
```

Comment:
10,4 are Magical numbers, you can use constants

Reply:
agreed